src/components/ErrorComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { translate } from '../plugins/i18n'
+
+const reload = () => {
+  window.location.reload()
+}
+
+const Oops = ({ t, errorType }) => (
```

Comment:
Oops, I forgot to rename this class x)
I agree but `Errorù is already a built-in class so I use this name for here.